tests_extractor.d

```diff
@@ -154,7 +156,10 @@ private:
         }
 
         // write the unittest block
-        outFile.write("unittest\n{\n");
+        if (config.betterCOutput)
+            outFile.writef("void unit_test_at_line_%s()\n{\n", line);
```

Comment:
I did it to avoid potential confusion with normal unit tests, whose function name is like: `__unittest_L11_C1`, but I guess it won't be that easy to mistake one for the other.

Reply:
done